packages/http/src/validator/validators/__tests__/headers.spec.ts

```diff
@@ -2,6 +2,7 @@ import { HttpParamStyles } from '@stoplight/types';
 import { query as registry } from '../../deserializers';
 import { HttpHeadersValidator } from '../headers';
 import * as validateAgainstSchemaModule from '../utils';
+import { assertRight } from '@stoplight/prism-core/src/utils/__tests__/utils';
 
 describe('HttpHeadersValidator', () => {
   const httpHeadersValidator = new HttpHeadersValidator(registry, 'header');
```

Comment:
so, for this PR validator classes like this - `HttpHeadersValidator` - stay classes and operate on `this` inside theirs `validate` behavior?

Reply:
Correct. The removing of the class would be a following PR, I do not want to bloat this too much or we'll end up like the Proxy PR that was huge and took ages to review and merge.